ooiservices/app/uframe/assetController.py

```diff
@@ -80,60 +80,22 @@ def _get_latlon(item):
     minutes = 0.0
     seconds = 0.0
     # scrub input
-    tmp, dir = _scrub_latlon(item)
+    tmp = _normalize(item)
     # process input and round result to _decimal places
     if tmp:
-        ds = tmp.split('^')
+        ds = tmp.split(' ')
         degrees = float(ds[0])
-        if "'" in tmp:
-            # minutes and seconds
-            part2 = str(ds[1])
-            g = part2.split("'")
-            minutes = float(g[0])
-            seconds = float(g[1])
-        else:
-            # minutes, no seconds
-            minutes = float(ds[1])
+        minutes = float(ds[1])
+        if len(ds) == 4:
+            seconds = float(ds[2])
         val = degrees +  ((minutes + (seconds/60.00000))/60.00000)
-        if dir:
-            if dir == 'W' or dir == 'S':
-                val = val*-1.0
+        if 'W' or 'S' in tmp:
```

Comment:
Heh, if you can find what needs to be hardened here then we'll tackle it.  TESTS!